### PR TITLE
dbuild: support menu and execution list

### PR DESCRIPTION
**Pull request by sunghan-chang**
1. Support "menu" option
  "./dbuild.sh menu" shows menu. User can choose an option
  after seeing list of available option.
  ======================================================
    "Select Board"
  ======================================================
    "1. artik053"
    "2. artik053s"
    "3. artik055s"
    "4. cy4390x"
    "5. qemu"
    "6. sidk_s5jt200"
    "x. EXIT"
  ======================================================

2. Support exection list
  User can put the list of execution as arguments like below:
  "./dbuild.sh configure artik053 hello"
  "./dbuild.sh distclean configure cy4390 tc build download all"

Signed-off-by: sunghan-chang <sh924.chang@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [8dad04433f21f87e68fba90c63e5dc89cdf4f115] -

**Comment by seinfra** [comment]
Target : [27e8c46f418d896d7df86542df961bb9acad071c] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [27e8c46f418d896d7df86542df961bb9acad071c] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [237520d9f9ffe1aafffedf6408f3236652005f22] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [237520d9f9ffe1aafffedf6408f3236652005f22] - Code Rule Check OK.

**Comment by sunghan-chang** [comment]
@samirsahu1947 @btheosam @dr-venkman Could you review my new guidance?

**Comment by seinfra** [comment]
Target : [d495d0f8b73ad634c4ce4fda59cf8a7d15466541] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [9ffa89cee44ceec86ae3abf17dd1411a125fe0d5] - Code Rule Check (C++) OK.

File: README.md [hunk 1]
```diff
@@ -35,7 +36,59 @@ TIZENRT_BASEDIR="$PWD"
 **Note**: To contribute in this community, you need to clone your forked private repository instead.  
           Github guides this by [working-with-forks](https://help.github.com/articles/working-with-forks).
 
-### 3. Configuration
+## How to build
```
Comment (davidfather) [inline README.md]: Numbering is required here.
For readability, naming convention is also required. For example, 
1. Installing Docker
2. Getting TizenRT source code
3. Building TizenRT
4. Programming

**Comment by seinfra** [comment]
Target : [38e24a2b14a8086507dc726a7600420655903b61] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [38e24a2b14a8086507dc726a7600420655903b61] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [70ebf57e7950d6c50f6bb5dc7d694a8e41408822] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [70ebf57e7950d6c50f6bb5dc7d694a8e41408822] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [1b397c395acec2e7238b5902ee47d4e6cb88c3d1] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [1b397c395acec2e7238b5902ee47d4e6cb88c3d1] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0b4d804917396f11019019a162ea209b32b2a8fc] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0b4d804917396f11019019a162ea209b32b2a8fc] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [25c3f2cb8044612c61362264f436af9f111b4448] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [25c3f2cb8044612c61362264f436af9f111b4448] - Code Rule Check OK.

**Pull request merged**